src/core/filesystem.cc

```diff
@@ -51,10 +51,17 @@
 #include <unistd.h>
 #include <cerrno>
 #include <fstream>
+
```

Comment:
remove this newline as well. all includes go together. You can fix some other things while you're here. The above ifdefs should move down to be just above the one you added. And then the lonely dirent.h above should be put in with the other includes. When you run formatter it will then order things correctly.

Reply:
working on it, thanks.

Reply:
done.